### PR TITLE
Add onClickScrim

### PR DESCRIPTION
**Pull request by kespinola**
### Notable Changes
- Add onClickScrim prop for TetheredDropdown. The function fires when you click the backdrop scrim of the dropdown.
- Fire onBlur of DropdownList when clicking scrim. This removes focus if not selected.

### CODE REVIEW

File: docs/components/examples/tetherPopup.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ var Example = React.createClass({
     		<${widgetName}
     			data={colors}
     			tetherPopup
+          onClickScrim={() => {console.log('clicking scrim')}}
```
Comment (dannyphillips) [inline docs/components/examples/tetherPopup.js:17]: you probably don't need this `console.log` anymore

**Comment by dannyphillips** [comment]
+1. Looks great!

File: dist/react-widgets.js [hunk 2]
```diff
@@ -1370,10 +1370,9 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	    var elementProps = omit(this.props, Object.keys(propTypes));
 	    var listProps = pick(this.props, Object.keys(List.propTypes));
-	    var popupProps = pick(this.props, Object.keys(_Popup2['default'].propTypes));
-
 	    var PopupComponent = tetherPopup ? _TetheredPopup2['default'] : _Popup2['default'];
 
+	    var popupProps = pick(this.props, Object.keys(PopupComponent.propTypes));
```
Comment (karismatic-megafauna) [inline dist/react-widgets.js:1375]: I believe this should be `popUp` 

I only make this point as below we have `dropDown`

I don't care either way, just consistency, and considering dropDown is already there, we should probably go with that.

File: dist/react-widgets.js [hunk 3]
```diff
@@ -3540,17 +3541,25 @@ return /******/ (function(modules) { // webpackBootstrap
...
+
+	    onBlur(e);
+	    onClickScrim && onClickScrim();
+	  },
+
+	  render: function render() {
+	    var _props2 = this.props;
+	    var className = _props2.className;
+	    var open = _props2.open;
+	    var dropUp = _props2.dropUp;
```
Comment (karismatic-megafauna) [inline dist/react-widgets.js:3557]: another case for `popUp`

File: lib/DropdownList.js [hunk 4]
```diff
@@ -230,10 +230,9 @@ var DropdownList = _react2['default'].createClass(babelHelpers.createDecoratedOb
 
     var elementProps = omit(this.props, Object.keys(propTypes));
     var listProps = pick(this.props, Object.keys(List.propTypes));
-    var popupProps = pick(this.props, Object.keys(_Popup2['default'].propTypes));
-
     var PopupComponent = tetherPopup ? _TetheredPopup2['default'] : _Popup2['default'];
```
Comment (karismatic-megafauna) [inline lib/DropdownList.js:233]: looks like `Popup` has been around for a while...

**Comment by karismatic-megafauna** [comment]
ahhh this is a fork of react-widgets, didn't notice that. My comments may be irrelevant.

**Comment by kespinola** [comment]
You got it. Just keeping the names he gave. Fyi anything in the lib folder is built files.

**Pull request merged**